streams/src/test/java/org/apache/kafka/streams/processor/internals/StandbyTaskTest.java

```diff
@@ -201,49 +201,37 @@ public void shouldTransitToRunningAfterInitialization() {
         task.initializeIfNeeded();
 
         assertEquals(RUNNING, task.state());
-
-        EasyMock.verify(stateManager);
     }
 
     @Test
     public void shouldThrowIfCommittingOnIllegalState() {
-        EasyMock.replay(stateManager);
         task = createStandbyTask();
         task.suspend();
         task.closeClean();
 
         assertThrows(IllegalStateException.class, task::prepareCommit);
     }
 
-
     @Test
     public void shouldAlwaysCheckpointStateIfEnforced() {
-        stateManager.flush();
-        EasyMock.expectLastCall().once();
-        stateManager.checkpoint();
-        EasyMock.expectLastCall().once();
-        EasyMock.expect(stateManager.changelogOffsets()).andReturn(Collections.emptyMap()).anyTimes();
-        EasyMock.replay(stateManager);
+        doNothing().when(stateManager).flush();
```

Comment:
can we verify that it was only called once? Right now even if it is invoked twice, Mockito won't throw an error. (please correct me if I am wrong)

Reply:
I have changed all `.expectLastCall().once()` statements into `verify` statements